.github/workflows/release.yml

```diff
@@ -114,16 +131,23 @@ jobs:
           if_true: ${{ github.event.inputs.name }}
           if_false: ${{ github.sha }}
 
-      - name: Wait for CI workflow (Multi-Arch image)
-        uses: fountainhead/action-wait-for-check@v1.0.0
-        id: wait-for-CI
+      - name: Wait for Multi-Arch build to succeed
+        uses: fountainhead/action-wait-for-check@v1.1.0
```

Comment:
why do you need to wait for the multiarch-build here?

Reply:
You shouldn't release if it wasn't successful